backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/hooks/StatusBranchHookExecutor.java

```diff
@@ -50,7 +53,7 @@ private Option<String> executeHookFor(String branchName) throws IOException, Int
       return Option.none();
     }
 
-    LOG.startTimer().debug(() -> "Executing machete-status-branch hook (${hookFilePath}) " +
+    LOG.debug(() -> "Executing machete-status-branch hook (${hookFilePath}) " +
```

Comment:
we do not care about the times now, right?

Reply:
We do! and `@Loggable` includes the execution time indeed, lemme put some log samples in the other comment...

Reply:
Great then!